docs/architecture/adr-008-app-caller-cbs/adr-008-app-caller-cbs.md

```diff
@@ -18,14 +18,15 @@ This setup worked well for off-chain users interacting with IBC applications.
 We are now seeing the desire for secondary applications (e.g. smart contracts, modules) to call into IBC apps as part of their state machine logic and then do some actions on the basis of the packet result. Or to receive a packet from IBC and do some logic upon receipt.
 
 Example Usecases:
-- Send an ICS-20 packet, and if it is successful, then send an ICA-packet to swap tokens on LP and return funds to sender
-- Execute some logic upon receipt of token transfer to a smart contract address
+
+* Send an ICS-20 packet, and if it is successful, then send an ICA-packet to swap tokens on LP and return funds to sender
```

Comment:
It is not a requirement and I don't remember making this change. However, I did run [mardownlint-cli](https://github.com/igorshubovych/markdownlint-cli) on the whole repo. It might be making this change when some criteria are met. It is also making sure the unordered list items are consistent per each markdown file. Markdown-cli uses the same rules in [here](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md004---unordered-list-style).

Reply:
I see, I think if we are going to format with `markdownlint-cli`, this should be a standard tool in the repo with some make command to run (ensure changes merged to main are formatted). Otherwise we can get a lot of diffs to review at random points when it is run. Opened #3543